packages/next/types/$$compiled.internal.d.ts

```diff
@@ -45,7 +45,7 @@ declare module 'react-server-dom-webpack/server.edge' {
         readonly name: string
         readonly async?: boolean
       }
-    },
+    } | null,
```

Comment:
This doesn't represent a valid type in React. Only an empty map is valid.

Reply:
My bad, I confused this with [`SSRModuleMap`](https://github.com/facebook/react/blob/ed966dac4a025fd37580e8197e5b271044ffbd9f/packages/react-server-dom-webpack/src/client/ReactFlightClientConfigBundlerWebpack.js#L33-L37) for some reason.

Reply:
It's because in the browser build it was actually optional but not in the new edge build.